tests/test_uci.py

```diff
@@ -0,0 +1,74 @@
+from chargemaster_parsers.parsers.uci import UCIChargeMasterParser
+from chargemaster_parsers.parsers import ChargeMasterEntry
+
+import pytest
+import requests
```

Comment:
Remove requests. It's causing the CI suite to fail and isn't needed

Reply:
fixed